spec/core/block/schemas/BlockMetaDTO.yml

```diff
@@ -16,9 +18,23 @@ properties:
     type: array
     items:
       $ref: "../../../schemas/Hash256.yml"
-  numTransactions:
+  totalTransactionsCount:
     type: integer
-    example: 0
-  numStatements:
+    description: >
+      Total number of [transactions](https://docs.symbolplatform.com/concepts/transaction.html)
+      present in this block, including *embedded* transactions
+      (i.e. transactions contained within aggregate transactions).
+    example: 1
+  transactionsCount:
+    type: integer
+    description: >
+      Number of [transactions](https://docs.symbolplatform.com/concepts/transaction.html)
+      present in this block. This does not count *embedded* transactions
+      (i.e. transactions contained within aggregate transactions).
+    example: 1
+  statementsCount:
     type: integer
+    description: >
+      Number of [transaction statements](https://docs.symbolplatform.com/concepts/receipt.html#transaction-statement)
+      present in this block.
```

Comment:
probably needs clarification on what "statements present in the block" means, just like transactions present means "confirmed"